bostrom_simulation/README.md

```diff
@@ -35,48 +35,83 @@ To optimize parameters for launching Bostrom.
+    delta_boot_inflation_rate = (1 - (vested_ratio/params['boot_bonded_share_target'])) * params['inflation_rate_change_annual']
+    delta_boot_inflation_rate = delta_boot_inflation_rate / params['timesteps_per_year']
+    boot_inflation_rate = previous_state['boot_inflation_rate'] + delta_boot_inflation_rate
+    if boot_inflation_rate > params['boot_inflation_max']:
+        boot_inflation_rate = params['boot_inflation_max']
+    elif boot_inflation_rate < params['boot_inflation_min']:
+        boot_inflation_rate = params['boot_inflation_min']
+    timestep_provision = (boot_supply * boot_inflation_rate) / params['timesteps_per_year']
+    return {'timestep_provision': math.floor(timestep_provision)}
+```
+
+
+
+### Simulation parameters
+
+- __*start_boot_supply*__ `(1e15)`
+- __*boot_inflation_rate_change*__  
+- __*boot_inflation_max*__  
+- __*boot_inflation_min*__  
+- __*boot_bonded_share_target*__  
+- __*boot_supply*__  
 
 ## Gift claim dynamics
  
 The addresses for gift are defined in the [research](https://github.com/Snedashkovsky/cybergift). This research [concludes](https://github.com/Snedashkovsky/cybergift#prize-to-be-the-first) 6M addresses for distribution of 70% of BOOT tokens. Further we need to model how this gifts can be claimed. 
 
-We need to derive *claim_rate* (by formula to define).
+We need to derive __*claim_rate*__ (by formula to define).
 
 Assumptions:
-- 100,000 agents will claim our gift. After that the gift will be activated (*start_agents_count* = 100_000)
-- it will take 100 days since genesis before the gift claiming process will be completed (*days_for_gift_activation* = 100)  
-
-  *days_for_gift_activation* is needed to refine days for gift activation to reach *start_agents_count*.
+- agents (__*agents_count_at_activation*__) will claim our gift. After that the gift will be activated 
+- it will take __*days_for_gift_activation*__ since genesis before the gift claiming process will be completed 
+- agents on the moment of gift activation will claim share of the gift (__*claimed_at_activation_share*__)
+- claim process can be prolonged on __*days_for_gift_full_claim*__ in case if __*agents_count_at_activation*__ will not reach the target goal by __*days_for_gift_activation*__.
+- __*liquid_boot_supply_share*__ - is share of BOOT supply that will be in circulation from the network launch
 
 After the modeling of claim dynamics, we can set baselines for adoption ("understanding network effects" chapter).
 
 ### Simulation parameters
 
-- *start_agents_count* `(10'000, 50'000, 100'000)`
-- *days_for_gift_activation* `(100, 150)`
-- *days_for_gift_full_claim* `(0, 360)` 
-- *claimed_at_activation_share* (`1, 0.5`)
+- __*agents_count_at_activation*__ `(10'000, 50'000, 100'000)`
+- __*days_for_gift_activation*__ `(100, 150)`
+- __*days_for_gift_full_claim*__ `(0, 360)` 
+- __*claimed_at_activation_share*__ `(1, 0.5)`
+- __*liquid_boot_supply_share*__ `(0.25)`
+- __*claimed_boot_supply*__ - ?
```

Comment:
finalize it

Reply:
Ales, if you know if claimed_boot_supply is useful for our modeling - please, help to define it. Otherwise it can be deleted